public/index.html

```diff
@@ -1,34 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <link
-      rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A==" crossorigin=""
-    />
-    <script
-      src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js" integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
-      crossorigin=""
-    ></script>
-    <link href="index.css" rel="stylesheet" />
-  </head>
-  <body>
-    <h1>Yelp...ington!</h1>
-    <!-- Title of the page-->
-
-    <div class="container">
-      <!--Container for the map and the  lis of the restaurants-->
-      <div id="restaurant-list">
-        <div class="scroll"  id="restaurant_list">
-        <!--Div for the restaurant list -->
-      </div>
-      </div>
-      <div id="map">
-        <!--MAP CONTAINER-->
-      </div>
-    </div>
-    <script src="index.js"></script>
-    <script>map()</script>
```

Comment:
Just a gentle push, but this is how you get map function to work in the index.html by _calling it,_ that probably is where you are going wrong for the functionality of the script in rest-page.html and the NOTWORKING.js....